forms/viewsets/form.py

```diff
@@ -1,17 +1,39 @@
 from django_filters.rest_framework import DjangoFilterBackend
-from rest_framework import viewsets
+from rest_framework import mixins, viewsets
 
-from forms.models import Answer, Form
-from forms.serializers.form import AnswerSerializer, FormSerializer
+from forms.models import Answer, Field, Form, Section
+from forms.serializers.form import (
+    AnswerSerializer,
+    FieldSerializer,
+    FormSerializer,
+    SectionSerializer,
+)
 
 
-class FormViewSet(viewsets.ReadOnlyModelViewSet):
+class FormViewSet(
+    mixins.RetrieveModelMixin,
+    mixins.UpdateModelMixin,
+    mixins.DestroyModelMixin,
```

Comment:
There should probably be some permission if form is template form. Because I suppose different groups can delete templates. (No need to do anything in this stage but somewhere could be note about this)

Reply:
Maybe samething concerning Update